.github/workflows/multiqc.yml

```diff
@@ -0,0 +1,37 @@
+name: multiqc
+on:
+  push:
+    paths:
+      - software/multiqc/**
+      - .github/workflows/multiqc.yml
+      - tests
+  pull_request:
+    paths:
+      - software/multiqc/**
+      - .github/workflows/multiqc.yml
+      - tests
+
+jobs:
+  ci_test:
+    runs-on: ubuntu-latest
+    env:
+      NXF_ANSI_LOG: false
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - name: Install Nextflow
+      run: |
+        export NXF_VER="20.07.1"
+        wget -qO- get.nextflow.io | bash
+        sudo mv nextflow /usr/local/bin/
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: "3.x"
+    - name: Install dependencies
+      run: python -m pip install --upgrade pip pytest-workflow
+
+    # Test the module
+    - run: pytest --tag multiqc --symlink --wt 2
```

Comment:
Please setup editorconfig in your editor to prevent the lack of a newline (GH is giving me a warning) https://editorconfig.org/

Reply:
Ah, the pesky tabs and spaces 😉